### ISSUE TITLE
cloudfiles cli cannot upload small files to gcs because of unconditional composite upload

### ISSUE THREAD

**Issue by ranlu** [seung-lab/cloud-files#93]
```
touch test.txt
cloudfiles cp test.txt gs://bucket/folder
```

produces the following error:

``` 
 File "/home/ranl/miniforge3/envs/cf/lib/python3.10/site-packages/cloudfiles/gcs.py", line 132, in composite_upload
    tree_depth = int(math.ceil(math.log2(num_parts) / math.log2(MAX_COMPOSITE_PARTS)))
ValueError: math domain error
```

**Comment by william-silversmith** [comment]
Thank you for reporting this!

**Comment by william-silversmith** [comment]
Fixed in 6cc4b3fa986c2294a7f2b71581b21dd53cf0be58
